packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -42,7 +42,7 @@ for PACKAGE in ${PACKAGES[@]}; do
         # It's a server, start the pm2 process
         echo "Starting ${PACKAGE}"
         yarn build
-        pm2 start --name $PACKAGE dist --wait-ready --listen-timeout 15000 --time
+        pm2 start --name $PACKAGE 'yarn start' --wait-ready --listen-timeout 15000 --time
```

Comment:
I believe this should be fine, but will deploy and test in a feature instance to ensure behaviour remains consistent with the way we initialized the daemon previously

Reply:
Could you look in particular at the prestart behaviour on restart? My only worry would be that it runs the prestart script when e.g. web-config-server crashes, meaning noticeable downtime before it comes back online.

If that's not the case, this is totally cool, and we can remove the `yarn build` above as the prestart hook should take care of that (plz check all packages though 🙏 )

Reply:
Excellent call about the `prestart` hook @edmofro ! I tested it and indeed it was rebuilding over every restart!

Got any clever ideas about how we might avoid this? My simplistic solution was to add a `start-post-build` script to each server that identical to the `start` one but due to it having a different name it avoids the `prestart` hook. I'm not the biggest fan of my implementation, nor the name, but was a little stumped to come up with anything better. Doesn't seem like node lets you skip the hook...

Reply:
Bummer...

One trick I've used in another spot is an environment variable
```
"postinstall": "test -n \"$SKIP_BUILD_INTERNAL_DEPENDENCIES\" || yarn build-internal-dependencies"
```

Which then allows
```
## run yarn without building internal dependencies, so we can cache that layer without code changes
## within internal dependencies invalidating it
RUN SKIP_BUILD_INTERNAL_DEPENDENCIES=true yarn install
```

We could use that same pattern here, with the result being something like
```
pm2 start --name $PACKAGE 'SKIP_PREBUILD=true yarn start' --wait-ready --listen-timeout 15000 --time
```

Reply:
I think I like that a bit more than my fix! Let's go with it 👍

Reply:
@edmofro apologies for the slow turnaround on this one. I tried testing the `test` command in my local bash shell and wasn't seeing anything so I thought maybe it wasn't working. Turns out it was, it just wasn't logging anything to the console 🤦 

Seems to work fine in my local now 🎉 will test on a feature instance before merging 👍